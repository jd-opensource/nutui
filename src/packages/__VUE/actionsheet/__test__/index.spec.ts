import { mount } from '@vue/test-utils';
import ActionSheet from '../index.vue';

test('');
