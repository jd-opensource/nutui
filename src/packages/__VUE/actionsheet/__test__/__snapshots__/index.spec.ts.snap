// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render sure actionsheet when use custom props 1`] = `
"<view class=\\"nut-actionsheet\\">
  <view>
    <transition-stub class=\\"\\">
      <view class=\\"nut-overlay\\" style=\\"animation-duration: 0.3s; z-index: 2000;\\"></view>
    </transition-stub>
    <transition-stub>
      <view class=\\"nut-popup round popup-bottom popclass\\" style=\\"z-index: 2000; animation-duration: 0.3s;\\">
        <view class=\\"nut-actionsheet-panel\\">
          <!--v-if-->
          <!--v-if-->
          <view class=\\"nut-actionsheet-menu\\">
            <view class=\\"nut-actionsheet-item\\" style=\\"color: rgb(26, 26, 26);\\">选项一<view class=\\"subdesc\\">描述信息</view>
            </view>
          </view>
          <!--v-if-->
        </view>
        <!--v-if-->
      </view>
    </transition-stub>
  </view>
</view>"
`;
